internal/controllers/machineset/machineset_controller.go

```diff
@@ -278,14 +277,14 @@ func (r *Reconciler) reconcile(ctx context.Context, cluster *clusterv1.Cluster,
 
 	var errs []error
 	for _, machine := range filteredMachines {
-		log.WithValues("machine", klog.KObj(machine))
+		log := log.WithValues("machine", klog.KObj(machine))
```

Comment:
```suggestion
		log := log.WithValues("Machine", klog.KObj(machine))
```
Should be uppercase because of the CR change, see https://github.com/kubernetes-sigs/cluster-api/pull/7075 / https://github.com/kubernetes-sigs/cluster-api/pull/7087 / https://github.com/kubernetes-sigs/controller-runtime/pull/1954

Just realized it's fixed on main, so that's more fyi and rebase should be enough

Reply:
rebased